### PR TITLE
Update logic on characteristics file presence and date check

### PR DESCRIPTION
**Pull request by jeanconn**
Remove date check on characteristics file and don't require characteristics file if dynamic aimpoint file present.

### CODE REVIEW

File: src/starcheck.pl [hunk 1]
```diff
@@ -715,11 +715,15 @@ sub json_obsids{
         else{
             $out .= "<A HREF=\"${att_report}\">[${red_font_start}NOT OK${font_stop}] Coordinate mismatch or error.</A>\n";
         }
-        if (check_characteristics_date($char_file, $date[0])){
-            $out .= "[OK] Characteristics file newer than 30 days\n\n";
-        }
-        else{
-            $out .= "[${red_font_start}NOT OK${font_stop}] Characteristics file older than 30 days\n\n";
+        # Only check that characteristics file is less than 30 days old if backstop starts before 01-Aug-2016
+        my $CHAR_DATE_CHECK_BEFORE = '2016:214:00:00:00.000';
```
Comment (jeanconn) [inline src/starcheck.pl]: I can obviously get rid of all of these "one off" variables, but it seemed like once this trend was started I might has well be consistent with it.

Comment (taldcroft) [inline src/starcheck.pl]: Yes, in this case I would prioritize consistency below avoiding useless one-offs.

File: src/starcheck.pl [hunk 2]
```diff
@@ -690,19 +690,18 @@ sub json_obsids{
...
         $out .= "Error.  dynamic aimpoint file not found. \n";
     }
-    # The attitude checks are not possible without the char file but are possible without the
-    # dynamic aimpoint file, so this is an if/else that controls doing the check only on the
-    # presence of the char_file.
-    if (not defined $char_file){
-        $out .= "Error.  Characteristics file not found. \n";
+    # The attitude checks are possible with either the characteristics file or the dynamic aimpoint file
+    # but not without both
+    if ((not defined $char_file) and (not defined $aimpoint_file)){
```
Comment (jeanconn) [inline src/starcheck.pl]: Is this more consistent with the plan?

Comment (taldcroft) [inline src/starcheck.pl]: Looks good to me.

**Comment by jeanconn** [comment]
I had tested this on AUG2916A before copying in the characteristics file.

**Pull request merged**